library/init/meta/has_reflect.lean

```diff
@@ -48,3 +48,9 @@ meta instance prod.reflect {α β : Type} [has_reflect α] [reflected α] [has_r
 
 meta instance pos.reflect : has_reflect pos
 | ⟨l, c⟩ := `(_)
+
+meta def reflected.to_format {α} {a : α} (h : reflected a) : format :=
+to_fmt (reflect a).to_expr
+
+meta instance {α} (a : α) : has_to_format (reflected a) :=
+⟨reflected.to_format⟩
```

Comment:
It was (very slightly) more effort to inline them, since the `reflected` instance wasn't available for type class inference in `reflect`. But using `to_fmt h.to_expr` avoids this.